pep-0517.txt

```diff
@@ -166,48 +166,71 @@ The build backend object is expected to have attributes which provide
 some or all of the following hooks. The common ``config_settings``
 argument is described after the individual hooks::
 
-  def get_build_requires(config_settings):
+  def build_sdist(source_directory, sdist_directory, config_settings):
+      ...
+
+MUST build an unpacked sdist from the ``source_directory`` and place it into the
```

Comment:
> In addition, we're already assuming (or we should be if we're not) that the build tool is going to be having logic to produce sdists, otherwise with setup.py gone, what is producing a sdist?

There's nowhere that's explicitly stated, and I'm not sure it's as obvious as you think. Personally, I find your argument compelling (with the proviso that I don't have a good feel whether it's pip or some other tool that should be the generic "sdist builder") but I do think it ought to be presented as an argument and agreed on. Could you post this point to distutils-sig as a thread of its own? I fear it'll get missed in the rest of the debate.